api/src/main/java/org/openmrs/module/owa/AppManager.java

```diff
@@ -128,4 +129,21 @@
 	 * @param appStoreUrl
 	 */
 	void setAppStoreUrl(String appStoreUrl);
+	
+	/**
+	 * Returns constructed string of missig modules and/or openmrs core with their required modules
```

Comment:
Can you add since annotations to these new methods?

Reply:
Thank you @dkayiwa , the next commit will reflect this good convention.